### DISCUSSION TITLE
Introduction to Github Activity Your First Commit is not refreshing to next step

### DISCUSSION THREAD

**Discussion by jgbutcher** [skills/.github#284] [Q&A]
I am trying to complete the Introduction to GitHub class.  I have completed the Activity your first branch and then just completed activity your first commit.  After I wait 20 seconds and refresh the page it is not moving to the next step in the tutorial.  Can you help me figure out how to get the  GitHub Actions to automatically update to the next step?

**Comment by sinsukehlab** [comment]
You should name your new branch "`my-first-branch`" (case-sensitive), for the workflows will check the branch name.

**Comment by Anjusha7** [comment]
it is still not updating. I created a new branch under the main with the same name. can anyone help ?

**Reply by sinsukehlab** [reply]
It seems that you haven't created a new branch. You can't create a new `main` branch or any branch with a name of an existing branch. Create a `my-first-branch` branch.
